docs/requirements.md

```diff
@@ -1,26 +1,26 @@
-# Requirements for a technical solution
+# Goals and Requirements for a Technical Solution
 
-## High level requirements: 
+## Goals
 
-To ensure that all users without discrimination will be able to reserve their rights, the solution shall:
+To ensure that all users without discrimination will be able to reserve TDM Rights, the solution shall:
 
-1. Rely on open standards and be interoperable.
-1. Rely on tools and formats widely supported by web crawlers.
-1. Be easy to implement, with no economic or technical barriers.
+1. Rely on open standards  - especially Web standards - and be interoperable.
```

Comment:
As there is no formal definition for what constitutes a "Web standard",  I would strongly *object* to this addition.

Reply:
That's not an important point, let's remove.